Command/README.md

```diff
@@ -0,0 +1,71 @@
+#  thin-edge.io Command plugin
+
+To use command on a device that runs thin-edge.io, a plugin of the operation plugin concept is used. The tedge_agent is checking for command operation and is triggering the particular plugin. You can use the Shell tab in device management to use a command. Please be aware of the power of such an feature. Somebody with access could start various shell commands. Its more designes as an idea on how the plugin mechanism works.
+
+## Requirements
+
+- Working thin-edge.io installation
+
+- Python3 and pip3 installation (will not work on python2)
+
+
+## Installation 
+
+1. Clone this repo on the thin-edge.io device
+2. run pip3 install -r requirements.txt from this Command directory
+3. Copy c8y_Command to the following directory "/etc/tedge/operations/c8y/"
+4. Copy c8y_Command.py to the following directory "/bin/"
+5. Make sure, that both files do have permissions for beeing executed by tedge_mapper ("chmod 777 FILENAME")
```

Comment:
Aside note: this recommendation highlight that thin-edge configuration and documentation are not clear.

* The mapper only reads the file  `/etc/tedge/operations/c8y/c8y_Command` and `chmod 644 c8y_Command` is enough.
* It would be more secure to prevent updates of the installed commands.  `chmod 555 c8y_Command.py` would be better.